### ISSUE TITLE
show fileGrp names exploiting full window width

### ISSUE THREAD

**Issue by bertsky** [hnesk/browse-ocrd#8]
OCR-D fileGrp USE strings can be quite long. In the drop-down list of the fileGrp dialog, longer names extruding from the pre-allocated width of the button are abbreviated with ellipses, which is a good thing. But why does the width of that button not utilise the full room that the parent panel provides?

![ocrd_browser_ellipsis](https://user-images.githubusercontent.com/38561704/94953975-af020100-04e8-11eb-824c-e0fd5ecc4e19.png)

**Comment by bertsky** [comment]
Also, it would be helpful if at least there was a tooltip text with the full name.

**Comment by hnesk** [comment]
I have removed the fixed width (120px) for the drop-down now and it seems to have no adverse effects. The reason it existed in the first place was to have space for more view options in that space, but that can be revisited, when more options are implemented.

As far as I can see, there is no possibility to have a tooltip per drop-down-option in GTK, explained here: https://mail.gnome.org/archives/gtk-list/2009-January/msg00057.html (although it's quite old , I think it's still correct)
At least I have added a tooltip for the Gtk.ComboBox widget itself, so it is possible to see the currently selected item unabridged.
